### PR TITLE
[RAINCATCH-728] Remove appforms

### PR DESCRIPTION
**Pull request by JameelB**
# What

- Update to remove all appforms related code. 
- Bumped user and mediator version. 

# JIRA
https://issues.jboss.org/browse/RAINCATCH-728

### CODE REVIEW

File: lib/app/admin/data-reset/data-demo/workorders.js [hunk 1]
```diff
@@ -5,21 +5,21 @@
...
     { id: "H1H76ij8r", workflowId: 'HJ8QkzOSH', assignee: 'HJ8QkzOSH', type: 'Job Order', title: 'Carpet in disrepair', status: 'New', startTimestamp: '2015-10-22T20:00:00Z', address: '860 Drake St, Vancouver, BC V6Z 2P2', location: [49.276903, -123.129645], summary: 'Please remove damaged kerb and SUPPLY AND FIX 1X DROP KERB CENTRE BN 125 X 150 cart away from site outside number 3.'},
-    { id: "BkuXajsIB", workflowId: 'SyVXyMuSr', assignee: null, type: 'Job Order', title: 'Sink in disrepair', status: 'New', startTimestamp: '2015-10-22T07:00:00Z', address: '3820 Oak St, Vancouver, BC V6H 2M5', location: [49.251362, -123.127070], summary: 'Please remove damaged kerb and SUPPLY AND FIX 1X DROP KERB CENTRE BN 125 X 150 cart away from site outside number 3.'}
+    { id: "BkuXajsIB", workflowId: 'HJ8QkzOSH', assignee: null, type: 'Job Order', title: 'Sink in disrepair', status: 'New', startTimestamp: '2015-10-22T07:00:00Z', address: '3820 Oak St, Vancouver, BC V6H 2M5', location: [49.251362, -123.127070], summary: 'Please remove damaged kerb and SUPPLY AND FIX 1X DROP KERB CENTRE BN 125 X 150 cart away from site outside number 3.'}
   ];
```
Comment (witmicko) [inline lib/app/admin/data-reset/data-demo/workorders.js]: not a major issue but could we add more static workorders to replace deleted appforms ones?

File: lib/app/workorder.js [hunk 2]
```diff
@@ -3,21 +3,21 @@
...
   { id: "H1H76ij8r", workflowId: 'HJ8QkzOSH', assignee: 'HJ8QkzOSH', type: 'Job Order', title: 'Carpet in disrepair', status: 'New', startTimestamp: '2015-10-22T20:00:00Z', address: '860 Drake St, Vancouver, BC V6Z 2P2', location: [49.276903, -123.129645], summary: 'Please remove damaged kerb and SUPPLY AND FIX 1X DROP KERB CENTRE BN 125 X 150 cart away from site outside number 3.'},
-  { id: "BkuXajsIB", workflowId: 'SyVXyMuSr', assignee: null, type: 'Job Order', title: 'Sink in disrepair', status: 'New', startTimestamp: '2015-10-22T07:00:00Z', address: '3820 Oak St, Vancouver, BC V6H 2M5', location: [49.251362, -123.127070], summary: 'Please remove damaged kerb and SUPPLY AND FIX 1X DROP KERB CENTRE BN 125 X 150 cart away from site outside number 3.'}
+  { id: "BkuXajsIB", workflowId: 'HJ8QkzOSH', assignee: null, type: 'Job Order', title: 'Sink in disrepair', status: 'New', startTimestamp: '2015-10-22T07:00:00Z', address: '3820 Oak St, Vancouver, BC V6H 2M5', location: [49.251362, -123.127070], summary: 'Please remove damaged kerb and SUPPLY AND FIX 1X DROP KERB CENTRE BN 125 X 150 cart away from site outside number 3.'}
 ];
```
Comment (witmicko) [inline lib/app/workorder.js]: not a major issue but could we add more static workorders to replace deleted appforms ones?

**Comment by JameelB** [comment]
@witmicko updated with additional static forms

File: lib/app/workorder.js [hunk 3]
```diff
@@ -3,21 +3,21 @@
...
   { id: "B1r71fOBr", workflowId: 'SyVXyMuSr', assignee: 'rkX1fdSH', type: 'Job Order', title: 'Driveway in disrepair', status: 'New', startTimestamp: '2015-10-22T07:00:00Z', address: '157 st, 163 Keefer St, Vancouver, BC V6A 1X4', location: [49.279490, -123.099947], summary: 'Please remove damaged kerb and SUPPLY AND FIX 1X DROP KERB CENTRE BN 125 X 150 cart away from site outside number 3.'},
-  { id: "HJ8QkzOSH", workflowId: 'SyVXyMuSr', assignee: 'rkX1fdSH', type: 'Job Order', title: 'Door in disrepair', status: 'New', startTimestamp: '2015-10-22T07:00:00Z', address: '2795 East Hastings St, Vancouver, BC V5K 1Z8', location: [49.281159, -123.047076], summary: 'Please remove damaged kerb and SUPPLY AND FIX 1X DROP KERB CENTRE BN 125 X 150 cart away from site outside number 3.'},
+  { id: "HJ8QkzOSH", workflowId: 'B1r71fOBr', assignee: 'rkX1fdSH', type: 'Job Order', title: 'Door in disrepair', status: 'New', startTimestamp: '2015-10-22T07:00:00Z', address: '2795 East Hastings St, Vancouver, BC V5K 1Z8', location: [49.281159, -123.047076], summary: 'Please remove damaged kerb and SUPPLY AND FIX 1X DROP KERB CENTRE BN 125 X 150 cart away from site outside number 3.'},
```
Comment (wtrocki) [inline lib/app/workorder.js]: Best to not mix id's of workflow and workorders.

File: lib/app/workorder.js [hunk 4]
```diff
@@ -3,21 +3,21 @@
...
   { id: "HJQTjsUr", workflowId: 'SyVXyMuSr', assignee: 'BJQm1G_BS', type: 'Job Order', title: 'Yard in disrepair', status: 'New', startTimestamp: '2015-10-22T15:00:00Z', address: '1780 E Broadway, Vancouver, BC V5N 1W3', location: [49.261782, -123.068705], summary: 'Please remove damaged kerb and SUPPLY AND FIX 1X DROP KERB CENTRE BN 125 X 150 cart away from site outside number 3.'},
-  { id: "Syx76jiUH", workflowId: 'SyVXyMuSr', assignee: 'BJQm1G_BS', type: 'Job Order', title: 'Sprinkler in disrepair', status: 'New', startTimestamp: '2015-10-22T07:00:00Z', address: '311 E Broadway, Vancouver, BC V5T 1W5', location: [49.262902, -123.098917], summary: 'Please remove damaged kerb and SUPPLY AND FIX 1X DROP KERB CENTRE BN 125 X 150 cart away from site outside number 3.'},
+  { id: "Syx76jiUH", workflowId: 'HJ8QkzOSH', assignee: 'BJQm1G_BS', type: 'Job Order', title: 'Sprinkler in disrepair', status: 'New', startTimestamp: '2015-10-22T07:00:00Z', address: '311 E Broadway, Vancouver, BC V5T 1W5', location: [49.262902, -123.098917], summary: 'Please remove damaged kerb and SUPPLY AND FIX 1X DROP KERB CENTRE BN 125 X 150 cart away from site outside number 3.'},
```
Comment (wtrocki) [inline lib/app/workorder.js]: [Question] Why we changing workflow id? Old one should be still valid.

**Comment by JameelB** [comment]
@wtrocki @witmicko update to use old initial seed data for workorders. Also made sure that the sample seed data for workorder reset is the same as the initial seed data for workorders

**Pull request merged**